Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutRequest.swift

```diff
@@ -10,11 +10,6 @@ import BraintreePayPal
 
 /// Options for the PayPal Checkout flow.
 @objcMembers public class BTPayPalNativeCheckoutRequest: BTPayPalCheckoutRequest {
-
-    // MARK: - Public Properties
-
-    /// Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
-    public var userAuthenticationEmail: String?
```

Comment:
just making sure, did we mean to remove the `userAuthenticationEmail` from `BTPayPalNativeCheckoutRequest`